addon/-private/system/store.js

```diff
@@ -281,6 +282,7 @@ Store = Service.extend({
     @return {DS.Model} record
   */
   createRecord(modelName, inputProperties) {
+    assert("You need to pass a type to the store's createRecord method", isPresent(modelName));
```

Comment:
It looks like it's not very consistent at the moment (see [this](https://git.io/v2NHM) and [this](https://git.io/v2NHH)), but I am in favor of `modelName` too, since it looks like the parameter for most method documentations is named `modelName` as well.